pkg/log/log.go

```diff
@@ -313,9 +313,23 @@ func IsInteractive() bool {
 
 // AddMaskedWord adds a new word to be redacted, only if longer than 5 chars
 func AddMaskedWord(word string) {
+	clean := strings.TrimSpace(word)
+
+	// only mask words longer than 5 chars (i.e. 'true' and 'false' will not be masked)
 	minCharsToMask := 5
-	if strings.TrimSpace(word) != "" && len(word) > minCharsToMask {
-		log.maskedWords = append(log.maskedWords, word)
+	if len(clean) < minCharsToMask {
+		return
+	}
+	log.maskedWords = append(log.maskedWords, clean)
+
+	// multi-line support
+	lines := strings.Split(clean, "\n")
```

Comment:
@andreafalzetti  Could you check in a windows system? The character for new lines is not the same so not sure there might be some issue there

Reply:
@ifbyol good point, I didn't remember about Win. I've tested now and it already works, I've done tests with PowerShell and cmd.exe but just in case I've added cross platform support with the last commit, to avoid any suprise in any edge case that I could not reproduce on Windows!